workers-site/redirects/oldDocs.js

```diff
@@ -89,4 +89,6 @@ export const oldDocsMap = new Map([
   ['/writing-workers/resource-limits', '/about/limits'],
   ['/writing-workers/storing-data', ''],
```

Comment:
BTW @signalnerve I noticed you added these blank redirects. Why is this here? if it's an archived article it will automatically redirect to the archived article?

Reply:
Hmm, good question! I don't remember 🙃  I think the redirect code goes to `/` if the redirect value is an empty string, right? Was maybe trying to just send these to `/`.... I honestly can't remember so I'm ok w/ you removing if you don't think we need them